### ISSUE TITLE
Syntax table names are fixed to blue

### ISSUE THREAD

**Issue by ygoe** [pawelsalawa/sqlitestudio#4760]
### Details
In the SQL editor, table names are in blue colour, always. There is no way to change that. I'm used to keywords being blue, so I've set that up. But then I noticed that the table names were already blue, so I looked for an option to change that. The only syntax class that had blue as its default colour was "valid objects". But changing that didn't change the colour of the table names. so it must be something else. It's also not reflected in the preview.

BTW, it would be nice if I could disable the bold style of keywords.

### Steps to reproduce
See screenshot.

### Operating system
Windows 10

### SQLiteStudio version
3.4.4

![grafik](https://user-images.githubusercontent.com/868315/231187557-5948a488-fe8e-4040-90a0-e1ecd68ed735.png)

**Comment by pawelsalawa** [comment]
Yeah, it seems that with these recent changes to color configuration, I messed up something. I've confirmed that changing this color does not work.

As for the bold/non-bold font, I've created #4762
